python/ray/autoscaler/v2/utils.py

```diff
@@ -349,58 +353,56 @@ def format(cls, data: ClusterStatus, verbose: bool = False) -> str:
         return formatted_output.strip()
 
     @staticmethod
-    def _node_usage_report(data: ClusterStatus, verbose: bool) -> str:
-        usage_by_node = {}
-        node_type_mapping = {}
-        idle_time_map = {}
+    def _node_usage_report(
```

Comment:
cc @ryanaoleary I refactored this function a bit:

1. Avoid passing the whole `ClusterStatus` to the function to make it more unit testable.
2. It's not necessary to pass `verbose` into this function.
3. Rename dictionaries to `....to....`.